R/zchunk_LA143.HDDCDD.R

```diff
@@ -1,25 +1,24 @@
 #' module_energy_LA143.HDDCDD
 #'
-#' Briefly describe what this chunk does.
+#' Reads in country level heating and cooling degree day data and returns GCAM region degree days via population weighting
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L143.HDDCDD_scen_R_Y}, \code{L143.HDDCDD_scen_RG3_Y}, \code{L143.HDDCDD_scen_ctry_Y}. The corresponding file in the
 #' original data system was \code{LA143.HDDCDD.R} (energy level1).
-#' @details Describe in detail what this chunk does.
+#' @details Population weights HDDCDD from country level to GCAm region
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_energy_LA143.HDDCDD_DISABLED <- function(command, ...) {
+#' @author RH April 2017
+
+module_energy_LA143.HDDCDD <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/iso_GCAM_regID",
              FILE = "energy/GIS_ctry",
-             "L101.Pop_thous_GCAM3_ctry_Y",
-             "i"))
+             FILE = "L101.Pop_thous_GCAM3_ctry_Y"))
```

Comment:
Inputs declared to driver need to include HDDCDD files below. We may need to discuss more how to handle this, but it's important that the driver has complete dependency information. This chunk can't go reading files on its own.